prime-dotnet-webapi/ViewModels/Health Authorities/HealthAuthorityViewModel.cs

```diff
@@ -1,4 +1,5 @@
 using System.Collections.Generic;
+using Prime.Models;
```

Comment:
Should a `ViewModel` class use/reference `Prime.Models`?

Reply:
Yes

Reply:
Right, should have done a text search prior to asking